### ISSUE TITLE
Updating database from 2.5.5 to 3.4.1 never continues from "INFO Queuing"

### ISSUE THREAD

**Issue by Larsss** [woocommerce/woocommerce#20356]
When updating woocommerce to newest version (from 2.5.5), the dialogue box asking to update database appears, I press it, but on page refresh it appears again as if it wasn't clicked.

Running on "Twenty Seventeen" theme, with all plugins disabled (except woocommerce base plugin).

This is the log from wc_db_updates:
2018-06-03T14:53:29+00:00 INFO Queuing 2.6.0 - wc_update_260_options
2018-06-03T14:53:29+00:00 INFO Queuing 2.6.0 - wc_update_260_termmeta
2018-06-03T14:53:29+00:00 INFO Queuing 2.6.0 - wc_update_260_zones
2018-06-03T14:53:29+00:00 INFO Queuing 2.6.0 - wc_update_260_zone_methods
2018-06-03T14:53:29+00:00 INFO Queuing 2.6.0 - wc_update_260_refunds
2018-06-03T14:53:29+00:00 INFO Queuing 2.6.0 - wc_update_260_db_version
2018-06-03T14:53:29+00:00 INFO Queuing 3.0.0 - wc_update_300_grouped_products
2018-06-03T14:53:29+00:00 INFO Queuing 3.0.0 - wc_update_300_settings
2018-06-03T14:53:29+00:00 INFO Queuing 3.0.0 - wc_update_300_product_visibility
2018-06-03T14:53:29+00:00 INFO Queuing 3.0.0 - wc_update_300_db_version
2018-06-03T14:53:29+00:00 INFO Queuing 3.1.0 - wc_update_310_downloadable_products
2018-06-03T14:53:29+00:00 INFO Queuing 3.1.0 - wc_update_310_old_comments
2018-06-03T14:53:29+00:00 INFO Queuing 3.1.0 - wc_update_310_db_version
2018-06-03T14:53:29+00:00 INFO Queuing 3.1.2 - wc_update_312_shop_manager_capabilities
2018-06-03T14:53:29+00:00 INFO Queuing 3.1.2 - wc_update_312_db_version
2018-06-03T14:53:29+00:00 INFO Queuing 3.2.0 - wc_update_320_mexican_states
2018-06-03T14:53:29+00:00 INFO Queuing 3.2.0 - wc_update_320_db_version
2018-06-03T14:53:29+00:00 INFO Queuing 3.3.0 - wc_update_330_image_options
2018-06-03T14:53:29+00:00 INFO Queuing 3.3.0 - wc_update_330_webhooks
2018-06-03T14:53:29+00:00 INFO Queuing 3.3.0 - wc_update_330_product_stock_status
2018-06-03T14:53:29+00:00 INFO Queuing 3.3.0 - wc_update_330_set_default_product_cat
2018-06-03T14:53:29+00:00 INFO Queuing 3.3.0 - wc_update_330_clear_transients
2018-06-03T14:53:29+00:00 INFO Queuing 3.3.0 - wc_update_330_set_paypal_sandbox_credentials
2018-06-03T14:53:29+00:00 INFO Queuing 3.3.0 - wc_update_330_db_version
2018-06-03T14:53:29+00:00 INFO Queuing 3.4.0 - wc_update_340_states
2018-06-03T14:53:29+00:00 INFO Queuing 3.4.0 - wc_update_340_state
2018-06-03T14:53:29+00:00 INFO Queuing 3.4.0 - wc_update_340_last_active
2018-06-03T14:53:29+00:00 INFO Queuing 3.4.0 - wc_update_340_db_version

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

Home URL: x
Site URL: x
WC Version: 3.4.1
Log Directory Writable: ✔
WP Version: 4.9.6
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: da_DK
External object cache: –

### Server Environment ###

Server Info: Apache/2
PHP Version: 5.6.36
PHP Post Max Size: 50 MB
PHP Time Limit: 580
PHP Max Input Vars: 2500
cURL Version: 7.59.0
OpenSSL/1.0.2k

SUHOSIN Installed: –
MySQL Version: 5.7.22
Max Upload Size: 50 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 2.5.5
WC Database Prefix: dlpWP_
Total databasestørrelse: 68.30MB
Database datastørrelse: 63.29MB
Database indexstørrelse: 5.01MB
dlpWP_woocommerce_sessions: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_api_keys: Data: 0.00MB + Index: 0.01MB
dlpWP_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_order_items: Data: 0.83MB + Index: 0.44MB
dlpWP_woocommerce_order_itemmeta: Data: 4.09MB + Index: 2.93MB
dlpWP_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB
dlpWP_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_log: Data: 0.00MB + Index: 0.00MB
dlpWP_commentmeta: Data: 0.06MB + Index: 0.00MB
dlpWP_comments: Data: 2.27MB + Index: 0.04MB
dlpWP_layerslider: Data: 0.03MB + Index: 0.00MB
dlpWP_links: Data: 0.00MB + Index: 0.00MB
dlpWP_options: Data: 0.68MB + Index: 0.02MB
dlpWP_postmeta: Data: 12.28MB + Index: 0.00MB
dlpWP_posts: Data: 7.97MB + Index: 0.00MB
dlpWP_revslider_css: Data: 0.02MB + Index: 0.00MB
dlpWP_revslider_layer_animations: Data: 0.00MB + Index: 0.00MB
dlpWP_revslider_navigations: Data: 0.00MB + Index: 0.00MB
dlpWP_revslider_settings: Data: 0.00MB + Index: 0.00MB
dlpWP_revslider_sliders: Data: 0.00MB + Index: 0.00MB
dlpWP_revslider_slides: Data: 0.00MB + Index: 0.00MB
dlpWP_revslider_static_slides: Data: 0.00MB + Index: 0.00MB
dlpWP_say_what_strings: Data: 0.00MB + Index: 0.00MB
dlpWP_termmeta: Data: 0.00MB + Index: 0.00MB
dlpWP_terms: Data: 0.00MB + Index: 0.00MB
dlpWP_term_relationships: Data: 0.00MB + Index: 0.00MB
dlpWP_term_taxonomy: Data: 0.00MB + Index: 0.00MB
dlpWP_usermeta: Data: 0.25MB + Index: 0.00MB
dlpWP_users: Data: 0.01MB + Index: 0.00MB
dlpWP_wc_download_log: Data: 0.00MB + Index: 0.00MB
dlpWP_wc_webhooks: Data: 0.00MB + Index: 0.00MB
dlpWP_woocommerce_termmeta: Data: 0.00MB + Index: 0.00MB
dlpWP_yoast_seo_links: Data: 0.00MB + Index: 0.00MB
dlpWP_yoast_seo_meta: Data: 0.00MB + Index: 0.00MB

### Post Type Counts ###

animated-columns: 4
attachment: 152
clients: 10
edge: 1
employees: 6
faq: 6
nav_menu_item: 1
page: 9
portfolio: 8
post: 16
product: 3
product_variation: 65
revision: 785
shop_coupon: 6
shop_order: 5273
shop_order_refund: 46
shop_webhook: 1
testimonial: 7

### Security ###

Secure connection (HTTPS): ❌
Hide errors from visitors: ❌

### Active Plugins (1) ###

WooCommerce: af Automattic – 3.4.1

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: DKK (DKK)
Currency Position: right_space
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)
ukategoriseret (ukategoriseret)


### WC Pages ###

Shop basisside: #5897 - /butik/
Kurv: #5898 - /kurv/
Checkout: #5899 - /kasse/
Min konto: ❌ Siden indeholder ikke en kortkode (shortcode).
Handelsbetingelser: #6893 - /home/handelsbetingelser/

### Theme ###

Name: Twenty Seventeen
Version: 1.6
Author URL: https://wordpress.org/
Child Theme: ❌ – Hvis du redigerer WooCommerce i et hovedtema
som du ikke selv har udviklet
anbefaler vi
at du anvender et undertema (child theme). Se: Hvordan du opretter et undertema

WooCommerce Support: ✔

### Templates ###

Overrides: –
`

```
</details>

**Comment by mikejolley** [comment]
Do you have issues with cron jobs running on this site?

If you run WP Crontrol, does trigger the update event manually help?

Finally, is there anything in the PHP error logs on this site? Could tell you if a process is erroring or timing out.

**Comment by mikejolley** [comment]
Without knowing more about the above we won't be able to offer guidance on this. It might be worth contacting the helpdesk so someone can login, once you've ruled out the cron jobs as the cause of those jobs not running.

https://woocommerce.com/contact-us/

As we cannot replicate this issue with the steps provided, I'm closing this issue for now.

Thanks

**Comment by Larsss** [comment]
Hi mikejolly, thanks for the reply, I haven't noticed any problems running other cron jobs on the site, I have tried running the update on two different servers, both with same issue. Triggering the event manually through WP Crontrol also does not help, it does say "Successfully executed the cron event wp_1_wc_updater_cron." but the problem persists. I have checked the PHP error logs, but there are no errors.

Thanks!
